knot.x/knotx/conf/routes/handlers/httpRepoConnectorHandler.conf

```diff
@@ -0,0 +1,34 @@
+clientOptions {
+  maxPoolSize = 1000
+  idleTimeout = 120 # seconds
+  tryUseCompression = true
+}
+
+clientDestination {
+  scheme = http
+  domain = 192.168.36.14
```

Comment:
As we talked offline, this should be replaced by some config.
Maybe you could use `host.docker.internal` for the time some better way will be available?
It will enable running that scenario locally.

Reply:
in progress...

Reply:
proposal completed. I like it :)